xen/arch/arm/ffa.c

```diff
@@ -364,6 +484,9 @@ static int __init ffa_init(void)
     printk(XENLOG_INFO "ARM FF-A Firmware version %u.%u\n",
            major_vers, minor_vers);
 
+    if ( !check_mandatory_feature(FFA_MSG_SEND_DIRECT_REQ_32) )
```

Comment:
in the arm64 case, don't we expect to have support for REQ_64 ?

Reply:
We are not using FFA_MSG_SEND_DIRECT_REQ_64 in the mediator, so things work even if the SP doesn't support FFA_MSG_SEND_DIRECT_REQ_64.